stripe/api_requestor.py

```diff
@@ -386,11 +415,17 @@ def request_raw(
     def _should_handle_code_as_error(self, rcode):
         return not 200 <= rcode < 300
 
-    def interpret_response(self, rbody, rcode, rheaders) -> StripeResponse:
+    def interpret_response(
+        self, rbody: object, rcode: int, rheaders: Mapping[str, str]
+    ) -> StripeResponse:
         try:
-            if hasattr(rbody, "decode"):
-                rbody = rbody.decode("utf-8")
-            resp = StripeResponse(rbody, rcode, rheaders)
+            resp = StripeResponse(
+                # TODO: should be able to remove this cast once self._client.request_with_retries
+                # returns a more specific type.
+                cast(Any, rbody).decode("utf-8"),
```

Comment:
Should we at least case to bytes?

Reply:
Done in [Address code review](https://github.com/stripe/stripe-python/pull/1088/commits/067ab1bfd6a74acd7cb4bb8c4f7e878a2d0e4661)